pkgs/tools/security/pass/default.nix

```diff
@@ -68,11 +68,11 @@ stdenv.mkDerivation rec {
     git
     gnupg
     gnused
-    procps
     pwgen
     tree
     which
-  ] ++ ifEnable x11Support [ dmenu xclip xdotool ]);
+  ] ++ stdenv.lib.optional stdenv.isLinux procps
```

Comment:
Forget the `[procps]` stuff - I always get confused with `optional` and `optionals`